Standards/scs-0210-v2-k8s-version-policy.md

```diff
@@ -0,0 +1,81 @@
+---
+title: SCS K8S Version Policy
+type: Standard
+status: Draft
+stabilized_at: 2023-08-16
+track: KaaS
+---
+
+## Introduction
+
+The Kubernetes project maintains multiple release versions including their patched versions.
+In the project, the three most recent minor releases are actively maintained, with a fourth
+version being in development. As soon as a new minor version is officially released,
+the oldest version is dropped out of the support period.
+Kubernetes supports its releases for around 14 months. 12 of these are the standard
+support period. The remaining 2 months are end-of-life support for things like:
+
+- CVEs (under the advisement of the Security Response Committee)
```

Comment:
What does "CVE" stand for?

Reply:
https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures

Reply:
Thank you for providing the link, I will put in a glossary section next week.